### DISCUSSION TITLE
Import geometry files

### DISCUSSION THREAD

**Discussion by kdg5424** [lululxvi/deepxde#1341] [Q&A]
Hello Lu Lu, Thanks for sharing great library. And I have two questions about geometry.

1. I want to apply DeepXDE to complex Geometry like vehicle, mixer, airfoil and ship. And if I have .stl or .stp files how could I import those files?

2. If it is impossible, how could I make a domain like airfoil, ship, and etc.?

Best regards

**Comment by lululxvi** [comment]
See FAQ Q: More details and examples about geometry. You can also use PointCloud https://deepxde.readthedocs.io/en/stable/modules/deepxde.geometry.html#deepxde.geometry.pointcloud.PointCloud
